acados/ocp_nlp/ocp_nlp_constraints_bgh.c

```diff
@@ -374,6 +374,10 @@ void ocp_nlp_constraints_bgh_dims_get(void *config_, void *dims_, const char *fi
     {
         ocp_nlp_constraints_bgh_get_nh(config_, dims_, value);
     }
+    else if (!strcmp(field, "nphi"))
```

Comment:
this is not needed any more and can be removed => modularity

Reply:
good catch, fixed.